pages/library/add.js

```diff
@@ -0,0 +1,142 @@
+import styled from "styled-components";
+import { useEffect, useState } from "react";
+import { useRef } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { v4 as uuidv4 } from "uuid";
+
+const StyledForm = styled.form`
+  display: grid;
+  gap: 0.2rem;
+  flex-direction: column;
+  grid-template-columns: 50% 50%;
+  margin-top: 16rem;
+  color: black;
+  justify-content: center;
+  align-items: center;
+  font-family: Georgia, "Times New Roman", Times, serif;
+`;
+
+const StyledLabel = styled.label`
+  text-align: right;
+`;
+
+const StyledInput = styled.input`
+  background-color: transparent;
+`;
```

Comment:
Again, a minor point. But there are inconsistent use of new lines between your styled components. It would be better to stick with one approach.